src/esp/assets/PTexMeshData.h

```diff
@@ -64,12 +62,26 @@ class PTexMeshData : public BaseMesh {
   virtual void uploadBuffersToGPU(bool forceReload = false) override;
   virtual Magnum::GL::Mesh* getMagnumGLMesh(int submeshID) override;
 
+  float exposure() const;
+  void setExposure(const float& val);
+
+  float gamma() const;
+  void setGamma(const float& val);
+
+  float saturation() const;
+  void setSaturation(const float& val);
```

Comment:
It would be great to add docstrings here, if nothing else at least a note about the inversion of gamma that's done inside.

Or to the shader. Or both.

Reply:
OK. I am working on it.